src/workers/commentMentionEmail.ts

```diff
@@ -6,6 +7,18 @@ import { baseNotificationEmailData, sendEmail, truncatePost } from '../common';
 import { Connection } from 'typeorm';
 import { getPostPermalink } from '../schema/posts';
 
+const getContent = (comment: string, username: string) => {
```

Comment:
Can you give this function a more explicit name? I'm not sure what you are trying to do